pkg/controller/workflow/reason.go

```diff
@@ -38,7 +38,7 @@ const (
 	ProjectNetworkPeerIsNotReadyInAtlas        ConditionReason = "ProjectNetworkPeerIsNotReadyInAtlas"
 	ProjectEncryptionAtRestReady               ConditionReason = "ProjectEncryptionAtRestReady"
 	ProjectCloudIntegrationsIsNotReadyInAtlas  ConditionReason = "ProjectCloudIntegrationsIsNotReadyInAtlas"
-	ProjectAuditingReady                       ConditionReason = "ProjectAuditingReady"
+	ProjectAuditLogsNotConfigured              ConditionReason = "ProjectAuditLogsNotConfigured"
```

Comment:
does this end up in status conditions? If so, this is effectively an API change.

Reply:
It's not a new status, just a more meaningful reason. Is it still a breaking change?

Reply:
+1

Reply:
This is removing `ProjectAuditingReady` in favor of `ProjectAuditLogsNotConfigured`.
If users rely on `ProjectAuditingReady` in their GitOps pipeline, they will be broken after this change.
Hence, we cannot simply remove reasons from status conditions, they are meant to be machine-readable.
`message` though can be improved.